doc/en/xdist.rst

```diff
@@ -1,197 +0,0 @@
-
-.. _`xdist`:
```

Comment:
I'm guessing this was all removed because it's sufficiently documented in pytest-xdist's docs?